lib/ansible/modules/network/meraki/meraki_snmp.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Kevin Breit (@kbreit) <kevin.breit@kevinbreit.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: meraki_snmp
+short_description: Manage organizations in the Meraki cloud
+version_added: "2.6"
+description:
+- Allows for management of SNMP settings for Meraki.
+notes:
+- More information about the Meraki API can be found at U(https://dashboard.meraki.com/api_docs).
+- Some of the options are likely only used for developers within Meraki.
+options:
+    state:
+        description:
+        - Specifies whether SNMP information should be queried or modified.
+        choices: ['query', 'present']
+        default: present
+    v2cEnabled:
```

Comment:
We tend to not add mixed-cased parameters, because YAML is case-sensitive and it makes it harder for people to remember how it should be properly written. I would go for **v2c_enabled** and **v3_authpass** and **perr_ips**.

Reply:
Updated to take this into account. Find and replace all is a bad idea.